arrow-site/docs/_includes/_nav.html

```diff
@@ -13,9 +13,22 @@
         <ul>
           {% for item in site.data.menu.nav%}
           {% if item.title != 'Github' %}
+          {% if item.title != "Training" %}
           <li class="nav-menu-item">
-            <a class="menu-links" href="{{ item.url }}" title="{{ item.title }}">{{ item.title }}</a>
+            <a class="menu-links" href="" title="{{ item.title }}">{{ item.title }}</a>
```

Comment:
```suggestion
            <a class="menu-links" href="{{ item.url }}" title="{{ item.title }}">{{ item.title }}</a>
```

To recover the `href` because it's empty now.